source/configure/install-mattermost-for-microsoft-teams-plugin.rst

```diff
@@ -1,141 +0,0 @@
-:orphan: :nosearch:
```

Comment:
I can't figure out how I missed this file, but it seems to simply be a duplicate?

Reply:
Great catch, @lieut-data! Yes, it's now a duplicate and this was a necessary clean up step. Thank you!